curp/src/server.rs

```diff
@@ -274,15 +273,11 @@ pub struct Protocol<C: Command + 'static, CE: CommandExecutor<C> + 'static> {
     cmd_executor: Arc<CE>,
     /// The channel to send synced command to sync manager
     sync_chan: MpscKeyBasedSender<C::K, SyncMessage<C>>,
-    /// Synced manager handler
-    sm_handle: JoinHandle<()>,
-    /// After sync tasks handle
-    after_sync_handle: Vec<JoinHandle<()>>,
     // TODO: clean up the board when the size is too large
     /// Cmd watch board for tracking the cmd sync results
     cmd_board: Arc<Mutex<HashMap<ProposeId, CmdBoardValue>>>,
-    /// trigger when commit_index changes
-    commit_trigger: UnboundedSender<()>,
```

Comment:
The commit_trigger is also moved into state to keep consistent with other triggers.